concordia/views.py

```diff
@@ -221,26 +221,23 @@ def post(self, *args, **kwargs):
             cmd = 'cp -r {0}/* {1}'.format('/concordia_images', collection_path)
             if os.WEXITSTATUS(os.system(cmd)) == 0:
                 os.system('rm -rf {0}'.format('/concordia_images/*'))
-            c = Collection.objects.create(title=name, slug=name.replace(" ", "-"), description=name)
-            count = 0
-            for root, dirs, files in os.walk(collection_path):
-                for filename in files:
-                    filename = os.path.join(root, filename)
-                    if True:
-                        count += 1
-                        title = '{0} asset {1}'.format(name, count)
-                        media_url = os.path.join(root, filename).replace(settings.MEDIA_ROOT, '')
+                c = Collection.objects.create(title=name, slug=name.replace(" ", "-"), description=name)
```

Comment:
Is it really the case that we want to create the collection after copying files to it? That would also allow replacing the currently duplicate code which creates a collection directory name in three places with a single method call on the `Collection` instance, which in addition to being more readable would avoid the possibility of bugs if the escaping convention every changes in the future but isn't repeated in every location.